src/audio/tdfb/tune/sof_bf_export.m

```diff
@@ -84,16 +84,14 @@
 	fprintf(1, 'No sof-ctl3 output file specified.\n');
 else
 	fprintf(1, 'Exporting to %s\n', bf.sofctl3_fn);
-	mkdir_check(bf.sofctl3_path);
```

Comment:
Note: The check of path for the new file is included to the sof_x_write() functions.